src/SuperVault.sol

```diff
@@ -206,6 +217,27 @@ contract SuperVault is BaseStrategy, ISuperVault {
         _updateSVData(superPositions, rebalanceArgs.finalSuperformIds);
     }
 
+    /// @inheritdoc ISuperVault
+    function forwardDustToPaymaster(address token_) external onlyVaultManager {
```

Comment:
This vault operates only with **asset** (which is equal among all vaults) and available from TokenizedStrategy.sol

Can remove token_ and the check below 

 if (IBaseForm(superform).getVaultAddress() == token_) {
                 revert CANNOT_FORWARD_SHARES();
}

--

        IERC20 token = IERC20(token_);

becomes

        IERC20 token = IERC20(address(asset);